src/scrot.c

```diff
@@ -40,8 +40,43 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 */
 
-#include "scrot.h"
+#include <sys/stat.h>
+#include <sys/wait.h>
+
 #include <assert.h>
+#include <err.h>
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#include <unistd.h>
+
+#include <Imlib2.h>
+#include <X11/Xlib.h>
+#include <X11/Xutil.h>
+#include <X11/extensions/Xcomposite.h>
+#include <X11/extensions/Xfixes.h>
+
+#include "imlib.h"
+#include "note.h"
+#include "options.h"
+#include "scrot.h"
+#include "slist.h"
+#include "util.h"
+
+size_t scrotHaveFileExtension(char const *, char **);
+Imlib_Image scrotGrabFocused(void);
+Imlib_Image scrotGrabAutoselect(void);
+Imlib_Image scrotGrabShotMulti(void);
+Imlib_Image scrotGrabStackWindows(void);
+Imlib_Image scrotGrabShot(void);
+void scrotCheckIfOverwriteFile(char **);
+void scrotExecApp(Imlib_Image, struct tm *, char *, char *);
+char *imPrintf(char *, struct tm *, char *, char *, Imlib_Image);
+Window scrotGetClientWindow(Display *, Window);
+Window scrotFindWindowByProperty(Display *, const Window, const Atom);
+Imlib_Image stalkImageConcat(ScrotList *, enum Direction const);
```

Comment:
These functions must be specified as `static` (declaration and definition).

Reply:
I made those static, and I also went through the other files and found a few more that could be made static. There were also some style fixes. Check the text of the 2nd commit of the latest force-push (I had to due to a conflict with master)